packages/@yoda/network/index.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+/**
+ * @module @yoda/network
+ * @description Provides classes to manage network functions on the device.
+ */
+
+var EventEmitter = require('events').EventEmitter
+
+class Network extends EventEmitter {
+  constructor (flora) {
```

Comment:
Why does the flora client not at the `Network`?

Reply:
Take a look at https://github.com/yodaos-project/yodart/blob/master/apps/cloudappclient/service.js.